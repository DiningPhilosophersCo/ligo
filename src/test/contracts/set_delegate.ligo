function main (const p: key_hash) : list(operation) is
  begin
    set_delegate(p) ;
    const dummy: list(operation) = nil;
  end with dummy

