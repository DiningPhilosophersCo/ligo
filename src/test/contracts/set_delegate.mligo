let main (p : key_hash) : operation list =
  let useless : operation = Tezos.set_delegate (Some p)
  in ([] : operation list)
