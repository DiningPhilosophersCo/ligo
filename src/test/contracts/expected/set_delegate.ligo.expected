function main (const p : key_hash) : list (operation) is
block {
  const unused : operation = set_delegate (Some (p));
  const dummy : list (operation) = nil
} with dummy
